internal/runtime/client/client_test.go

```diff
@@ -269,3 +273,279 @@ func TestURLForExtension(t *testing.T) {
 		})
 	}
 }
+
+func TestClient_CallExtension(t *testing.T) {
+	g := NewWithT(t)
+
+	testHostPort := "127.0.0.1:9090"
+
+	fpFail := runtimev1.FailurePolicyFail
+	fpIgnore := runtimev1.FailurePolicyIgnore
+
+	validExtensionHandlerWithFailPolicy := runtimev1.ExtensionConfig{
```

Comment:
Note: if creating an ExtensionConfig is ~ a one liner, should we change the test case parameter to `registeredExtensionHandlers` and create the registry on the fly in the for loop? I think this would be easier as there is more useful info directly visible in the table test case